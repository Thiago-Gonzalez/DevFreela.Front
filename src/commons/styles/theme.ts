import { COBALT, BLACK_SQUEEZE } from "../constants/colors";

const theme = {
    components: {
        MuiCssBaseline: {
            styleOverrides: {
                body: {
                    margin: 0,
                    padding: 0
                }
            }
        }
    },
    palette: {
        primary: {
            main: COBALT,
        },
        background: {
            default: BLACK_SQUEEZE,
        }
    }
}

export default theme;