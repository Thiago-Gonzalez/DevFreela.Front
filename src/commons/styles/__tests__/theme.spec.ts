import theme from "../theme"


describe("[commons] Theme", () => {
    it('should snapshot by theme object', () => {
        expect(theme).toMatchSnapshot();
    })
})