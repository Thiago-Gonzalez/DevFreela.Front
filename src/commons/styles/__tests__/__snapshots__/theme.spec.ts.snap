// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[commons] Theme should snapshot by theme object 1`] = `
Object {
  "components": Object {
    "MuiCssBaseline": Object {
      "styleOverrides": "
        @font-face {
          font-family: 'Raleway';
          font-style: normal;
          font-display: swap;
          font-weight: 400;
          src: local('Raleway'), local('Raleway-Regular'), url(raleway-regular-webfont.woff) format('woff2');
          unicodeRange: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF;
        }

        body {
          margin: 0;
          padding: 0;
          overflow: hidden;
        }
      ",
    },
  },
  "palette": Object {
    "background": Object {
      "default": "#F9FBFD",
    },
    "primary": Object {
      "main": "#0041BE",
    },
  },
  "typography": Object {
    "fontFamily": "Raleway",
  },
}
`;
